jupyter/jupyter-web-app/base/config-map.yaml

```diff
@@ -107,10 +107,19 @@ data:
         #       class:
         #         value: {none}
         readOnly: false
-      extraResources:
-        # Extra Resource Limits for user's Notebook
-        # e.x. "{'nvidia.com/gpu': 2}"
-        value: "{}"
+      gpus:
+        # Number of GPUs to be assigned to the Notebook Container
+        value:
+          # values: "none", "1", "2", "4", "8"
+          num: "none"
+          # Determines what the UI will show and send to the backend
+          vendors:
+            - limitsKey: "nvidia.com/gpu"
+              uiName: "NVIDIA"
+            - limitsKey: "amd.com/gpu"
```

Comment:
Do we want AMD here? On GCP I don't think we have AMD GPUs and to my knowledge they aren't compatible with TF.

I suspect we will eventually need to create overlays for different platforms. I suspect NVIDIA is supported on most platforms/clouds.